copter/source/docs/autotune.rst

```diff
@@ -107,6 +139,7 @@ Additional Notes
 -   AutoTune is sometimes unable to find a good tune for frames with very soft vibration dampening of the autopilot or very flexible arms.
```

Comment:
we can probably remove this line, "AutoTune is sometimes unable to find a good tune for frames with very soft vibration dampening of the autopilot or very flexible arms." now that we've got the failure reason section at the top.

Reply:
will do